Sources/AppKitBackend/BaseWidget.swift

```diff
@@ -0,0 +1,113 @@
+import AppKit
+
+public class BaseWidget: NSView {
```

Comment:
I believe that you're not actually using `BaseWidget` anywhere? If so this should be removed.